src/sentry/replays/endpoints/organization_replay_events_meta.py

```diff
@@ -22,6 +25,17 @@ class OrganizationReplayEventsMetaEndpoint(OrganizationEventsV2EndpointBase):
 
     private = True
 
+    def get_field_list(self, organization: Organization, request: Request) -> Sequence[str]:
+        return [
+            "id",
+            "error.value",
+            "timestamp",
+            "error.type",
+            "issue.id",
+            "issue",
+            "group.id",
```

Comment:
What is `group.id`, is it different than `issue.id`?

Reply:
yeah, they're different. It's something like this: `/organizations/${org-slug}/issues/${group.id}/events/${event.id}`

I also want to grab the group shortID which is something like `POKEDEX-3NN`